### PR TITLE
Business Hours: Update front-end of the description list

### PR DESCRIPTION
**Pull request by thomasguillot**
Reduce the size taken by the Business Hours on a page.

#### Changes proposed in this Pull Request:

* Make sure parent element clears floats
* Have the terms and descriptions on the same line
* Terms elements should be bold to differentiate them from the descriptions (in case theme doesn't already style this)
* Overwrite description's default margin
* Add small right margin to terms

#### Testing instructions:

* Add a Business Hours block to any post and page
* Notice the new style in the editor when block is unfocussed
* Notice the new style on the front-end

__Before:__

1 before
_Twenty Nineteen / Shoreditch_

__After:__

2 after
_Twenty Nineteen / Shoreditch_

#### Proposed changelog entry for your changes:

* Business Hours: Update front-end of the description list

### CODE REVIEW

**Comment by jeherve** [comment]
> Tests failed: D28309-code

Since you are adding new files, after you merge this PR you will need to add the new files to the diff manually to get the tests to pass.

**Review by jeherve — APPROVED** [review]
It does look better imo. 👍 This should be good to merge once it's design approved.

**Comment by shaunandrews** [comment]
Much nicer without all that extraneous whitespace. Can you remove the repetitious "From" as well?

**Comment by thomasguillot** [comment]
I can have a look at the "From ... to ..." but probably in a different PR as it touches a different portion of the code

**Comment by jeherve** [comment]
> probably in a different PR as it touches a different portion of the code

In this case this should be good to merge. I've unblocked the PR so you should be able to merge, but as I mentioned above, you'll need a bit of manual work to port your changes over to WordPress.com in D28309-code this time:

> Since you are adding new files, after you merge this PR you will need to add the new files to the diff manually to get the tests to pass.

**Comment by thomasguillot** [comment]
Thank you @jeherve

**Pull request merged**
